### PR TITLE
Create details page

### LINKED ISSUE #6: [3pt]  Create details page(Melaku)
- When the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) with its full description (skip the "Rotate image" button)
- In the details page, the user can click the "Reserve" button
- When the user clicks the "Add item" link in the navigation panel

### LINKED ISSUE #9: [2pt] Create the redux store (group task)
- Create the redux store

### LINKED ISSUE #10: [2pt] Create the doctor reducer (group task)
- Implement a redux reducer for the Doctor.

### PR DESCRIPTION
**Pull request by Melaku05**
- [x] When the user selects a specific item, they can see the details page with its full description (skip the "Rotate image" button)
- [x] In the details page, the user can click the "Reserve" button
- [x] When the user clicks the "Add item" link in the navigation panel

### CODE REVIEW

File: src/components/Details.js [hunk 1]
```diff
@@ -1,7 +1,72 @@
...
-const Details = () => (
-  <div>Details</div>
-);
+const Details = () => {
+  const doctors = useSelector((state) => state.doctor);
+  const { id } = useParams();
+  const doctor = doctors.data[id - 1];
+  return (
+    <div className="grid grid-flow-row">
+      <div className="lg:grid lg:grid-cols-3  flex flex-col gap-2 mt-20  ">
```
Comment (aboongm) [inline src/components/Details.js:12]: The details page works. Good works.

File: src/components/Details.js [hunk 2]
```diff
@@ -1,7 +1,72 @@
 import React from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { BiLeftArrow } from 'react-icons/bi';
```
Comment (aboongm) [inline src/components/Details.js:4]: Good choice using react-icon!

**Review by aboongm — APPROVED** [review]
Hi @Melaku05, 
- [x] The details page works. 👏 👏 👏 

Your milestone is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

request_changes_1

## Highlights

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Details.js (hunk 2 above)
Comment (Melaku05) [reply src/components/Details.js:4]: Thank you for reviewing our code brother.

**Pull request merged**